src/modules/BootSupportCheck.rb

```diff
@@ -179,6 +179,9 @@ def efi?
     end
 
     def check_zipl_part
+# storage-ng
+=begin
```

Comment:
Well, translating this code to storage-ng should be quite easy. Any reason to disable it instead of updating it?